src/compton2/Compton2.hh

```diff
@@ -0,0 +1,197 @@
+//----------------------------------*-C++-*-----------------------------------//
+/*!
```

Comment:
Use standard 100-col format (I know - we adopted this format after you started this branch. sorry.)

Reply:
should be easy enough to find a place to copy-paste from